docker/Dockerfile

```diff
@@ -1,26 +1,19 @@
-FROM python:3.9-slim-buster as parent
+FROM python:3.9-slim-bullseye as parent
 
 ENV PYTHONUNBUFFERED=1
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN echo "Install base packages" && apt-get update \
     && apt-get install -y --no-install-recommends \
-        apt-transport-https \
```

Comment:
This is no longer needed as it's included in apt by default these days: https://packages.debian.org/bullseye/apt-transport-https